src/system/OptionRow/OptionRow.js

```diff
@@ -107,7 +107,7 @@ const OptionRow = ( {
 				{subTitle && <Text sx={{ mb: 1, color: 'muted' }}>{subTitle}</Text>}
 				{body && <Text sx={{ mb: 0 }}>{body}</Text>}
 			</Box>
-			<Box>{meta ? meta : <MdArrowForward size={24} />}</Box>
+			<Box>{meta ? meta : ! disabled && <MdArrowForward size={24} />}</Box>
```

Comment:
I wonder if we should not couple the `disabled` prop to the `meta` prop this way. It may be valuable to display the arrow even when the row is disabled - to indicate that if that option was enabled, it would take you somewhere, for example.

Reply:
That's a good point. Perhaps it would be better to just allow removing the arrow separately? E.g. by explicitly meta to `false` – `<OptionRow meta={false}  />`? What do you think @brunobasto ?

```suggestion
			{false !== meta && (
				<Box>{meta ? meta : <MdArrowForward size={24} />}</Box>
			)}
```

Reply:
Done in [0b9ae26](https://github.com/Automattic/vip-design-system/pull/63/commits/0b9ae2644b0110f5827261d9aa00a89f3e51698f)